src/raftstore/store/config.rs

```diff
@@ -208,6 +212,13 @@ impl Config {
                                 lease));
         }
 
+        if self.accelerate_campaign_reserved_ticks >= self.raft_election_timeout_ticks {
```

Comment:
If `randomized_election_timeout == election_timeout`, setting `accelerate_campaign_reserved_ticks` to `raft_election_timeout_ticks` doesn't take effect to accelerate campaign. IMO it's better to put strict limit on `accelerate_campaign_reserved_ticks` to avoid mis-configuration.